### ISSUE TITLE
hpcviewer sometimes erroneously hides the last metric

### ISSUE THREAD

**Issue by jmellorcrummey** [HPCToolkit/hpcviewer.e4#51]
Using the formula facility, I created a new GPU time metric by summing kernel time, sync time, and copy time. It displayed as the last metric. However, when I hid some of the other metrics by deselecting them, it hid the last metric column even though the metric selector shows it to be shown.

**Comment by laksono** [comment]
Which version did you use? Which branch and on which platform?

**Comment by jmellorcrummey** [comment]
I used the ppc64le version that I copied from the tar files in your home directory on gpu.cs that you pointed Marty at. I used it with Java 8 on a P9 system at Sandia on an export-controlled application. I can't supply the database. 

I used the tar file from your directory because of Sandia's firewall permissions that prevent me from downloading maven with spack. After I downloaded maven with wget, I ran into failures trying to run maven when it tried to download pom files, which it couldn't because of Sandia's firewall permissions. This led me to use your prebuilt copy.

**Comment by laksono** [comment]
This is strange. I tried on llnl.cs and it works just fine.
I'll try to build again the viewer with Java 8 (I used Java 11 on gpu.cs).

**Comment by jmellorcrummey** [comment]
Hmm. I did some more experimentation after shutting down and restarting hpcviewer at Sandia. I tried with OpenJDK and with ibm-java. I can't replicate the problem in the bottom-up view from a clean start. I can't precisely state what operations I did in my prior session that preceded the problem. Without that, there isn't a good way to debug this. 

Let's mark it as a low priority issue until I see it again and can provide a reproducer.
